bin/services/wait-until-up

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+while ! ./bin/services/are-up
+do
+  echo "Waiting for services..."
+  sleep 0.01
```

Comment:
The time in the sleep is actually basically irrelevant, it could probably go. Connecting via docker limits this to about 2 per second anyway.